test/typing/plain_files/orm/issue_9340.py

```diff
@@ -28,7 +29,7 @@ class UserComment(Message):
     __mapper_args__ = {
         "polymorphic_identity": "user_comment",
     }
-    username: Mapped[str] = mapped_column(nullable=True)
+    username: Mapped[Optional[str]] = mapped_column(nullable=True)
```

Comment:
I think this is something that sqlalchemy wanted to support. cc @zzzeek

Reply:
yeah why are we changing this?    this is testing somehting pretty specific